src/pages/cartmanagement/cartmanagement.css

```diff
@@ -156,7 +156,7 @@
   }
   
     .imganddetails{
-      width:100%;
+      width:90%;
       height:300px;
       margin:1rem auto;
```

Comment:
try using rem if value is more than 10pixels also ```img-details``` or ```imgDetails``` can be used here as the classname is not readable